src/picongpu/include/plugins/radiation/Radiation.kernel

```diff
@@ -234,145 +238,144 @@ void kernelRadiationParticles(ParBox pb,
                  * LATER: can this be optimized?
                  */
                 int saveParticleAt = -1;
-                PMACC_AUTO(par,(*frame)[linearThreadIdx]);
 
-                /* if radiation is not calculated for all particles
-                 * but not via the gamma filter, check which particles
-                 * have to be used for radiation calculation
-                 */
+                /* if particle is not moving we skip all calculations */
+                if( !((particle_momentumNow == particle_momentumOld) &&
+                      (particle_momentumNow == vector_32::zero())) )
```

Comment:
Please remove the second part of this test. One **can have radiation even if the particle has temporally a speed of zero**. 
There is only no radiation if there is no acceleration:
```
 particle_momentumNow == particle_momentumOld
```

Additionally `particle_momentumNow == vector_32::zero()` should never be true since we give usually a temperature to the gas.